include/quicly/cc.h

```diff
@@ -52,12 +59,28 @@ typedef struct st_quicly_cc_t {
      * Packet number indicating end of recovery period, if in recovery.
      */
     uint64_t recovery_end;
+    /**
+     * Congestion window at the end of slow start.
+     */
+    uint32_t cwnd_exiting_slow_start;
+    /**
+     * Minimum congestion window during the connection.
+     */
+    uint32_t cwnd_minimum;
+    /**
+     * Maximum congestion window during the connection.
+     */
+    uint32_t cwnd_maximum;
+    /**
+     * Congestion controller in use.
+     */
+    quicly_congestion_controller_t controller;
```

Comment:
How about moving this to the top of the struct? I would assume that in the future this value would affect how the other members are used / organized.